hash_version.sh

```diff
@@ -1,8 +1,10 @@
-#!/bin/bash
+#!/bin/sh
 
-GIT_HASH=`sh ./calc_hash.sh`;
+GIT_HASH=$(sh ./calc_hash.sh)
 
+# NOTICE:
+# don't use sed -i, because it's not portable between Linux, OS X and some BSDs
 rm -f ver
-sed -b "s/NO_GIT_COMMIT_HASH_DEFINED/$GIT_HASH$1/" renderdoc/data/resource.h > ver && mv ver renderdoc/data/resource.h
-sed -b "s/NO_GIT_COMMIT_HASH_DEFINED/$GIT_HASH$1/" renderdocui/Properties/AssemblyInfo.cs > ver && mv ver renderdocui/Properties/AssemblyInfo.cs
+sed "s/NO_GIT_COMMIT_HASH_DEFINED/$GIT_HASH$1/" renderdoc/data/resource.h > ver && mv ver renderdoc/data/resource.h
```

Comment:
ok, i've now loggen on my Windows and I see that sed in MSYS2 has `-b` flag. Dunno about Cygwin. sed in bash in Ubuntu in Windows 10 probably doesn't have `-b` because it uses packages from Ubuntu repo.